pkg/v1/static/static_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package static
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/types"
+	"github.com/google/go-containerregistry/pkg/v1/validate"
+)
+
+func TestNewLayer(t *testing.T) {
+	b := []byte(strings.Repeat(".", 10))
+	l := NewLayer(b, types.OCILayer)
+
+	// This does basically nothing.
+	if err := validate.Layer(l, validate.Fast); err != nil {
```

Comment:
Why fast? You can drop half this test if you do standard.

Reply:
```
    static_test.go:33: gzip: invalid header
```

Reply:
Of course :(

Reply:
We should maybe fix that to actually look at media types (for +gzip)

Reply:
Do you want to do that? I'm fine just manually validating in this test for now.